Emby.Server.Implementations/Data/SqliteUserRepository.cs

```diff
@@ -1,85 +1,86 @@
-using System;
```

Comment:
you seem to have again broken line endings here, I had to switch to "no whitespace diff" mode to read this file